roles/ocp_on_libvirt/tasks/libvirt_host_destroy.yml

```diff
@@ -1,20 +1,31 @@
 ---
-- name: "Does node exist"
+- name: "Does node exist - {{ host['name'] }}"
+  community.libvirt.virt:
+    command: list_vms
+  register: _ool_vms_list
+
+- name: "Set node_not_exists"
```

Comment:
If keeping this task (i.e. not merging this one and the next one), should we rename it to `Set ool_node_exists`, instead?